src/app/api/api_v1/endpoints/project.py

```diff
@@ -38,6 +42,40 @@ async def read_project(
     return result
 
 
+@router.get("/{id}/generate", response_description="Zip file containing the generated code")
+async def generate_project(
+        *,
+        db: AsyncSession = Depends(deps.get_async_db),
+        _id: int
+) -> Any:
+    """"
+    Retrieve generated code of the project
```

Comment:
```suggestion
    Generate and retrieve code of the project
```
This sounds too much like it assumes pre-generated code.